src/services/webclient.ts

```diff
@@ -1147,6 +1153,9 @@ export class WebClientService {
             // Remove all pending promises
             this.wireMessageFutures.clear();
 
+            // Stop pending timeouts
+            this.timeoutService.cancelAll();
```

Comment:
Still missing at the other places where `this.wireMessageFutures.clear` is called. :)

Reply:
Ah, that's what you meant :slightly_smiling_face:

Reply:
Should be done now.